azure.go

```diff
@@ -329,12 +392,11 @@ func (b *azureAuthBackend) getAzureSettings(ctx context.Context, config *azureCo
 
 func ConfigurationFromName(name string) (cloud.Configuration, error) {
 	configs := map[string]cloud.Configuration{
-		"AZURECHINACLOUD":        cloud.AzureChina,
-		"AZUREPUBLICCLOUD":       cloud.AzurePublic,
-		"AZUREUSGOVERNMENTCLOUD": cloud.AzureGovernment,
+		"AzureChinaCloud":        cloud.AzureChina,
+		"AzurePublicCloud":       cloud.AzurePublic,
+		"AzureUSGovernmentCloud": cloud.AzureGovernment,
```

Comment:
Wondering if we changed normalizing the input here so that things match up with casing in `GetGraphURI()`? I sort of liked that users could supply it in any casing instead of requiring them to get the pascal case right. Any chance we could continue to allow users to supply it in any casing by modifying `GetGraphURI()` instead?

Reply:
That makes sense; I updated `GetGraphURI()` to check on the uppercased string instead of the pascal case. This allows the user to supply the `environment` field whatever way they prefer